java/test/jmri/jmrix/can/cbus/swing/eventtable/CbusEventTablePaneTest.java

```diff
@@ -293,28 +290,15 @@ private void initFrame(){
     
     }
     
-    @Test
-    public void testGetHelpTarget(){
-        assertEquals( helpTarget, panel.getHelpTarget(), "help target");
-    }
-
-    @Test
-    public void testGetTitle(){
-        assertEquals( title, panel.getTitle(),"title");
-    }
-    
     @TempDir 
     protected Path tempDir;
 
     private CanSystemConnectionMemo memo; 
     private TrafficControllerScaffold tcis; 
     private CbusConfigurationManager configM;
-
-    protected JmriPanel panel = null;
-    protected String helpTarget = null;
-    protected String title = null;
     
     @BeforeEach
+    @Override
     public void setUp() {
         // super.setUp();
```

Comment:
I don't know about this, so I'm asking just to check:
Now when CbusEventTablePaneTest extends jmri.util.swing.JmriPanelTest, it maybe should call the super method?

Reply:
Thanks for the reminder, I meant to look at the super method on this.
I found a few nullpointers with the dispose method in CbusEventTablePane, resolved by moving a few things to constructor.